<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FocusPositionSettingDialog</class>
 <widget class="QDialog" name="FocusPositionSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MANUAL_FOCUSPOSITION_002</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icon.qrc">
    <normaloff>:/escan2_app.ico</normaloff>:/escan2_app.ico</iconset>
  </property>
  <widget class="QPushButton" name="OK_pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ES2U_MAIN_046</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ES2U_MAIN_047</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="FocusPosition_label">
      <property name="text">
       <string>MANUAL_FOCUSPOSITION_002</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="FocusPosition_colon_label">
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GammaSlider" name="FocusPosition_slider">
      <property name="minimum">
       <number>-20</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GammaDoubleSpinBox" name="FocusPosition_spinBox">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-2.000000000000000</double>
      </property>
      <property name="maximum">
       <double>6.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaSlider</class>
   <extends>QSlider</extends>
   <header>gammaslider.h</header>
   <slots>
    <slot>setGammaSlider(double)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>GammaDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>gammadoublespinbox.h</header>
   <slots>
    <slot>setGamma(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OK_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>FocusPositionSettingDialog</receiver>
   <slot>PushButtonAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>FocusPositionSettingDialog</receiver>
   <slot>PushButtonAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FocusPosition_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FocusPosition_spinBox</receiver>
   <slot>setGamma(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FocusPosition_spinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FocusPosition_slider</receiver>
   <slot>setGammaSlider(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PushButtonAction()</slot>
 </slots>
</ui>
